cli/checkpoint/checkpoint.go

```diff
@@ -0,0 +1,40 @@
+package checkpoint
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+
+	"github.com/tarantool/tt/cli/cmdcontext"
+)
+
+// CmdOpts contains flags for managing checkpoint files commands.
+// Used for commands tt cat and tt play, which are checkpoint files commands.
+type CmdOpts struct {
```

Comment:
Maybe just `Opts` or `CheckpointOpts`? Up to you.

Reply:
Done with `Opts`

Now in cmd.cat it looks like `var catFlags = checkpoint.Opts{...}`

Reply:
Fixed with `math.MaxUint64`

Now it looks like struct with a comment in front of it:
```
// catFlags contains flags for cat command.
// Initialized with default values at creation.
var catFlags = checkpoint.Opts{
	From:       0,
	To:         math.MaxUint64,
	Space:      nil,
	Format:     "yaml",
	Replica:    nil,
	ShowSystem: false,
}
```